tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_Minmod.cpp

```diff
@@ -552,8 +550,31 @@ void test_lambda_pin_troubled_cell_tvb_correction(
   const double m2 = 2.0;
 
   const auto input = [&mesh]() noexcept {
-    const auto coords = logical_coordinates(mesh);
-    return DataVector{10.0 * step_function(get<0>(coords))};
+    // We want a function that is somewhat step-like. But for the test to work
+    // using GaussLobatto _and_ Gauss points, we need to use a representable
+    // polynomial function.
+    //
+    // So, we find here the smooth cubic that equals 10*step_function on a
+    // 4-point LGL mesh. This has the form,
+    //   f(x) = 5 + A x + B x^3
+    // subject to the constraint f(x) = 10*step_function(x) on the LGL mesh,
+    // i.e.,
+    //   f(1) = 5 + A + B = 10
+    //   f(1/sqrt(5)) = 5 + A / sqrt(5) + B / sqrt(5)^3 = 10
+    // giving the coeffs:
+    //   A = 5 (sqrt(5)^3 - 1) / 4
+    //   B = 1 - A = 25 (1 - sqrt(5)) / 4
+    //
+    // Finally, we scale around the mean by a factor just under 1, so make sure
```

Comment:
`so as to make sure` (or something else more English)

Reply:
Went with "to make sure".